src/module/apps/sidebar/chat-log.ts

```diff
@@ -72,6 +73,22 @@ export class ChatLogPF2e extends ChatLog<ChatMessagePF2e> {
                     }
                 }
             }
+
+            const revertDamageButton = htmlClosest<HTMLButtonElement>(target, "button.revert-damage");
```

Comment:
Use a `data-action` selector for this: we try to keep styling and recent listeners on separate selectors.

Reply:
Done